demos/wasi/hello-world-rust/k8s.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   name: hello-world-wasi-rust
 data:
   myval: "cool stuff"
+  lorem_ipsum.txt: |
```

Comment:
Nit: [Bacon ipsum](https://baconipsum.com) or [hipster ipsum](https://hipsum.co) will provide more entertainment value 😆. You could also use my old haiku I wrote up

Reply:
give me bacon